DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Administrator/DeactivateAdminConfirmation.cshtml

```diff
@@ -0,0 +1,20 @@
+﻿@{
+  ViewData["Title"] = "User deactivated";
+}
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 id="page-heading" class="nhsuk-heading-l">
```

Comment:
I don't believe the id is actually getting used, so it can safely be removed.

Reply:
Id is removed